### PR TITLE
Blog post for 2.1.5

### PR DESCRIPTION
**Pull request by nicosomb**

### CODE REVIEW

File: content/posts/20161121-wallabag-215.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+date: 2016-11-21 11:00:00+00:00
+slug: wallabag-215
+title: [Release] wallabag 2.1.5
+tags: v2, release
+
+If your server runs with PHP 5.5.x, it's time to update your wallabag installation: 2.1.5 is here!
```
Comment (tcitworld) [inline content/posts/20161121-wallabag-215.md]: Add a sentence to say 2.1.4 wasn't 5.5 compatible.

Comment (nicosomb) [inline content/posts/20161121-wallabag-215.md]: done.

Comment (tcitworld) [inline content/posts/20161121-wallabag-215.md]: ?

Comment (nicosomb) [inline content/posts/20161121-wallabag-215.md]: https://github.com/wallabag/wallabag.org/commits/master ;-)

**Pull request merged**